examples/ogbn_train.py

```diff
@@ -34,17 +34,22 @@
     help='Root directory of dataset.',
 )
 parser.add_argument(
-    "--model",
-    type=str.lower,
-    default='SGFormer',
-    choices=['sage', 'gat', 'sgformer'],
-    help="Model used for training",
+    '--gnn_choice',
+    type=str,
+    default='graph_transformer_sgformer',
+    choices=[
+        'gat', 'sage', 'graph_transformer_sgformer',
```

Comment:
please switch this to "SGFormer" and "Polynormer" and then below, keep a hardcoded list of which choices are graph transformer, then you can do args.gnn_choice = "graph_transformer_"+ args.gnn_choice.
please also mirror all of this to the cugraph examples including the appropriate changes to the forward pass (imagine sampling worked, since in the future it will and we want this to be the base reference).